src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -62,9 +89,9 @@ private void calculatePid(double speed) {
 
         // We'll eventually need an acceleration
         double ffv = feedforward.calculate(speed, 0);
-        SmartDashboard.putNumber("PID Calculation", pidv);
-        SmartDashboard.putNumber("FF Calculation", ffv);
-        SmartDashboard.putNumber("PID + FF", pidv + ffv);
+        pidWidget.getEntry().setNumber(pidv);
```

Comment:
Since you use `getEntry()` everywhere and don't mess with the widget afterwards, it might make sense to store the *entries* instead of the widgets. If you want to be really fancy you could even try to store them in a map.